lms/djangoapps/courseware/module_render.py

```diff
@@ -278,15 +279,16 @@ def get_module_for_descriptor(user, request, descriptor, field_data_cache, cours
         grade_bucket_type=grade_bucket_type,
         static_asset_path=static_asset_path,
         user_location=user_location,
-        request_token=request_token(request),
+        request_token=xblock_request_token(request),
+        disable_staff_debug_info=disable_staff_debug_info,
     )
 
 
-def get_module_system_for_user(user, field_data_cache,
+def get_module_system_for_user(user, field_data_cache,  # TODO  # pylint: disable=too-many-statements
```

Comment:
It's supposed to be for the pylint warning.  Pylint is complaining that this method is waaay too big, which is completely true.  We really should refactor it at some point.  So the TODO was added along with disabling the pylint warning.